tests/phpunit/tests/blocks/wpGetBlockPatterns.php

```diff
@@ -1,35 +1,55 @@
 <?php
 /**
- * Tests for _wp_get_block_patterns.
+ * Tests for WP_Theme::get_block_patterns.
  *
  * @package WordPress
  * @subpackage Blocks
  * @since 6.4.0
  *
  * @group blocks
  *
- * @covers ::_wp_get_block_patterns
+ * @covers WP_Theme::get_block_patterns
  */
-class Tests_Blocks_WpGetBlockPatterns extends WP_UnitTestCase {
+class Tests_Blocks_WPThemeGetBlockPatterns extends WP_UnitTestCase {
```

Comment:
Maybe better to rename this to `Tests_Blocks_WPThemeGetBlockPatterns`, as this is now covering a method on the `WP_Theme` class? The test file should then be moved to the `tests/phpunit/tests/theme` directory.

IMO that makes more sense since other theme specific tests, including those related to blocks, are in that folder.